<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CMainForm</class>
 <widget class="QMainWindow" name="CMainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtModelViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_3">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>150</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>颜色</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_5">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>19</y>
         <width>161</width>
         <height>121</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="colorVLayout"/>
      </widget>
     </widget>
    </item>
    <item row="1" column="0" rowspan="2">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>250</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>OpenGL</string>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>19</y>
         <width>160</width>
         <height>221</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="openGLVLayout">
        <item>
         <widget class="QCheckBox" name="chkAntiAliasing">
          <property name="text">
           <string>反锯齿</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkGourand">
          <property name="text">
           <string>Gourand着色</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkLighting">
          <property name="text">
           <string>光照</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="chkShowBackFace">
          <property name="text">
           <string>显示背面</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radPoint">
          <property name="text">
           <string>点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radLine">
          <property name="text">
           <string>线</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radFace">
          <property name="text">
           <string>面</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="radFaceLine">
          <property name="text">
           <string>面线混合</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1" rowspan="4">
     <layout class="QVBoxLayout" name="drawWidgetLayout"/>
    </item>
    <item row="3" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="minimumSize">
       <size>
        <width>180</width>
        <height>100</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>180</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>动画</string>
      </property>
      <widget class="QCheckBox" name="chkRotation">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>20</y>
         <width>156</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>旋转</string>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_open"/>
    <addaction name="action_save"/>
    <addaction name="action_saveAs"/>
    <addaction name="action_close"/>
    <addaction name="separator"/>
    <addaction name="action_exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>操作</string>
    </property>
    <addaction name="action_Segmentation"/>
    <addaction name="separator"/>
    <addaction name="action_FillHole"/>
    <addaction name="action_Smooth"/>
    <addaction name="action_Simplify"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_open"/>
   <addaction name="action_save"/>
   <addaction name="action_saveAs"/>
   <addaction name="action_close"/>
   <addaction name="action_exit"/>
   <addaction name="action_about"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_open">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/open.png</normaloff>:/QtModelViewer/images/open.png</iconset>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </action>
  <action name="action_save">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/save.png</normaloff>:/QtModelViewer/images/save.png</iconset>
   </property>
   <property name="text">
    <string>保存</string>
   </property>
  </action>
  <action name="action_saveAs">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/saveAs.png</normaloff>:/QtModelViewer/images/saveAs.png</iconset>
   </property>
   <property name="text">
    <string>另存为</string>
   </property>
  </action>
  <action name="action_exit">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/quit.png</normaloff>:/QtModelViewer/images/quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
   <property name="toolTip">
    <string>退出</string>
   </property>
  </action>
  <action name="action_close">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/close.png</normaloff>:/QtModelViewer/images/close.png</iconset>
   </property>
   <property name="text">
    <string>关闭</string>
   </property>
   <property name="toolTip">
    <string>关闭</string>
   </property>
  </action>
  <action name="action_holeFill">
   <property name="text">
    <string>补洞</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="qtmodelviewer.qrc">
     <normaloff>:/QtModelViewer/images/about.png</normaloff>:/QtModelViewer/images/about.png</iconset>
   </property>
   <property name="text">
    <string>关于</string>
   </property>
  </action>
  <action name="action_Segmentation">
   <property name="text">
    <string>分割</string>
   </property>
  </action>
  <action name="action_FillHole">
   <property name="text">
    <string>补洞</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+H</string>
   </property>
  </action>
  <action name="action_Smooth">
   <property name="text">
    <string>光顺</string>
   </property>
  </action>
  <action name="action_Simplify">
   <property name="text">
    <string>细分</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtmodelviewer.qrc"/>
 </resources>
 <connections/>
</ui>
